src/Microsoft.AspNetCore.OData/Extensions/ODataRouteBuilderExtensions.cs

```diff
@@ -39,23 +38,7 @@ public static IRouteBuilder SetDefaultQuerySettings(this IRouteBuilder builder,
                 throw Error.ArgumentNull("builder");
             }
```

Comment:
Now I understand the reason for this change. I still feel extension method may not be right. Just a shared static method with a normal parameter may be better.

Reply:
Extension method, we can call the method using Fluent way.

builder.Fun1().Func2().Func3()....

with static method. we can't do like this.

---
In reply to: [375990865](https://github.com/OData/WebApi/pull/2035#discussion_r375990865) [](ancestors = 375990865)